### ISSUE TITLE
Driver Inattention Detection using Image Processing

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#505]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Driver Inattention Detection using Image Processing</br>
:red_circle: **Aim** : The aim is to identify whether the driver is loosing attention or not using deep learning methods.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/zeyad1mashhour/driver-inattention-detection-dataset</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by tushtithakur** [comment]
Hi , I'm excited to contribute to this project. Could you please assign me? Looking forward to getting started! @abhisheks008

Full name : Tushti Thakur
GitHub Profile Link : 
Email ID : https://github.com/tushtithakur
Approach for this Project : Implement different deep learning algorithms using the dataset, evaluate it and compare performance.
What is your participant role? GSSoC 2024

**Comment by abhisheks008** [comment]
Hi @tushtithakur wait for the induction session to complete by today evening, after that issues will be assigned to the contributors.

**Comment by Subhranil2004** [comment]
Hi @abhisheks008 , I am willing to contribute to this issue! Please assign me to it.

- Full name : Subhranil Nandy
- GitHub Profile Link : https://github.com/Subhranil2004
- Email ID : iamsubhranil.nandy@gmail.com
- Approach for this Project : I would like to do some EDA, some image preprocessing and apply different DL(CNN) techniques for model creation and evaluation.
- What is your participant role? GSSoC 2024 Contributor

**Comment by abhisheks008** [comment]
Hi @tushtithakur can you elaborate your approach here? What are the models you are planning to use?

@Subhranil2004 one issue at a time please. Let others have their chance too.

**Comment by tushtithakur** [comment]
@abhisheks008 Sure! Here's the approach I will be following:-
1. Initially, I will perform exploratory data analysis and visualise the data to gain more insights 
2. I will perform Data preprocessing and normalise the data.
3.Will use pretrained models such as VGG, ResNet and fine tune them on the given dataset.
4.Will use deep learning models such as CNN, RNN, CapsNet
5. Train the models on training set and evaluate them on the test.
6. Comparing the performance of the different models used.

**Comment by gitsofaryan** [comment]
I'm ready to work on this, I have an idea- if the driver is drowsing we can detect it and call or buzzer a sound to break its attention and to prevent accidents. Under gssoc24

**Comment by abhisheks008** [comment]
Issue assigned to you @tushtithakur.

Hi @gitsofaryan nice to have you here. This issue is assigned to Tushti based on FCFS. You can check out other issues present here in this repo.

**Comment by VoonnaHarsha** [comment]
@abhisheks008 
Hi , I'm excited to contribute to this project. Could you please assign me? Looking forward to getting started! 

Full name : Voonna Harsha Vardhan
GitHub Profile Link : https://github.com/VoonnaHarsha
Email ID : harshavoonna@gmail.com

What is your participant role? GSSoC 2024

**Comment by abhisheks008** [comment]
> @abhisheks008 Hi , I'm excited to contribute to this project. Could you please assign me? Looking forward to getting started!
> 
> Full name : Voonna Harsha Vardhan GitHub Profile Link : https://github.com/VoonnaHarsha Email ID : [harshavoonna@gmail.com](mailto:harshavoonna@gmail.com)
> 
> What is your participant role? GSSoC 2024

This issue is already assigned to a contributor. Please check out the issues which are having the label `Up for Grabs`.
